services/configuration/CHANGELOG.md

```diff
@@ -7,10 +7,35 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 ### Breaking changes
 
-- `CONFIG_SERVICE` constant has been renamed to `CONFIGURATION_SERVICE_ID`.
+- Most types renamed to avoid [stuttering] (#496):
 
-- `CONFIG_PROPOSE_MESSAGE_ID` and `CONFIG_VOTE_MESSAGE_ID` constants are no
-  longer public.
+  - `ConfigurationService` to `Service`
+  - `ConfigurationServiceFactory` to `ServiceFactory`
+  - `TxConfigPropose` to `Propose`
+  - `TxConfigVote` to `Vote`
+  - `ConfigurationSchema` to `Schema`
+  - `StorageValueConfigProposeData` to `ProposeData`
+  - `CONFIG_SERVICE` constant to `SERVICE_ID`
+
+  Check the crate documentation for more details.
+
+[stuttering]: https://doc.rust-lang.org/1.0.0/style/style/naming/README.html#avoid-redundant-prefixes-[rfc-356]
```

Comment:
FWIW, it's not an *authoritative* style guide, and the recent libraries guidelines intentionally do not include this rule: https://rust-lang-nursery.github.io/api-guidelines/naming.html.

So, looks like the state of the art is "it is unclear, which is better, io::Error or IoError, use your own judgment": https://github.com/rust-lang-nursery/futures-rfcs/pull/2#issuecomment-364271680

Reply:
Yeah, I intended to use a link primarily to explain the term, not to appeal to authority. As a side note, `Error`, `Result` and some other names are special, because they shadow types included into *every* scope.